packages/requests/src/lib/requests-result.spec.ts

```diff
@@ -426,6 +434,152 @@ describe('requests result', () => {
             "isError": false,
             "isLoading": false,
             "isSuccess": true,
+            "responseData": undefined,
+            "status": "success",
+            "successfulRequestsCount": 1,
+          },
+        ],
+      ]
+    `);
+  });
+
+  it('should cache response data', () => {
+    jest.useFakeTimers().setSystemTime(new Date('2020-01-01'));
+
+    const { state, config } = createState(withEntities<Todo>());
+
+    const store = new Store({ state, config, name: 'todos' });
+
+    const entities$ = store.pipe(
```

Comment:
Doing this isn't make sense for what you did. The whole purpose of this feature is NOT to use the joinRequestResult operator.